src/ui/buttons/buttons.jsx

```diff
@@ -170,19 +170,15 @@ export function Buttons(props : ButtonsProps) : ElementNode {
                         personalization={ personalization }
                     /> : null
             }
-
-            {
-                (fundingSources.indexOf(FUNDING.CARD) !== -1)
-                    ? <div id="card-fields-container" class="card-fields-container" />
-                    : null
-            }
-
-            {
-                (layout === BUTTON_LAYOUT.VERTICAL && fundingSources.indexOf(FUNDING.CARD) !== -1)
-                    ? <PoweredByPayPal
-                        locale={ locale }
-                    /> : null
-            }
+            
+            {fundingSources.indexOf(FUNDING.CARD) !== -1 ? (
+                <div class="card-fields-with-logo">
```

Comment:
Please add `card-fields-with-logo` and `card-fields-container` to the [class constants file](https://github.com/paypal/paypal-checkout-components/blob/master/src/constants/class.js)

Reply:
Sure